### DISCUSSION TITLE
Use a map with a more detailed rendering

### DISCUSSION THREAD

**Discussion by AndreasHoffmann2** [streetcomplete/StreetComplete#4072] [Ideas]
I have been using SC for a while now and really like completing the information. But there is one gap when considering POIs like benches, bicycle parkings or barriers: as soon as the POI exists, SC asks for the details. But there is no way to check if the POI exists at all.

So in practice, when I encounter one of these, I check SC to see if there is information missing. And if not, I do sometimes switch to Vespucci to check if the POI exists at all. And pretty often, this POI is still missing entirely. But I don't know how many POIs I did NOT add to OSM, because I did not switch to Vespucci to do this check.

I would really like it if SC could at least give me chance to see that there is something missing. I know that SC is not meant to be a full blown OSM-editor. That would be way too complex. But perhaps, there is a way to achieve both.

One way could be the usage of a more detailed rendering for the map. A rendering that displays the POIs, that can be completed with SC. Is this even possible?

When standing in front of such a POI, I could then see that it is still missing and add it using Vespucci.

One step further would be to allow the addition of these POIs: A long-touch could open a menu to choose the type. And after the creation, the normal tasks would kick in and ask for all the details.

What do you think? Does this go too far?

**Comment by Helium314** [comment]
This probably could be done with layers, see #2461. But this may take a while until it is finished.
